lib/openxml/shapes/text.rb

```diff
@@ -0,0 +1,73 @@
+require "openxml/elements/shape_non_visual"
+require "openxml/elements/text_body"
+require "openxml/elements/paragraph"
+require "openxml/elements/run"
+require "openxml/elements/text"
+require "openxml/elements/body_properties"
+require "openxml/elements/shape_properties"
+require "openxml/elements/nonvisual_drawing_properties"
+require "openxml/elements/nonvisual_shape_drawing_properties"
+require "openxml/elements/nonvisual_properties"
+
+module OpenXml
+  module Shapes
+    class Text
+      attr_accessor :text
+      private :text=
+
+      def initialize(text)
+        self.text = text
+      end
+
+      def nonvisual_shape_property
+        @nonvisual_shape_property ||= OpenXml::Elements::ShapeNonVisual.new.tap {|nv_shape_property|
+          nv_shape_property << OpenXml::Pptx::Elements::NonvisualDrawingProperties.new.tap { |nvdp|
+            nvdp.id = object_id
+            nvdp.name = "TextBox 1"
+          }
+          nv_shape_property << OpenXml::Pptx::Elements::NonvisualShapeDrawingProperties.new
+          nv_shape_property << OpenXml::Pptx::Elements::NonvisualProperties.new
+        }
+      end
+
+      def shape_property
+        @shape_property = OpenXml::Elements::ShapeProperties.new.tap { |sp|
+          sp << OpenXml::DrawingML::Elements::TransformEffect.new.tap { |effect|
+            effect << OpenXml::DrawingML::Elements::Offset.new.tap { |offset|
+              offset.x = 0
+              offset.y = 0
+            }
+            effect << OpenXml::DrawingML::Elements::Extension.new.tap { |extension|
+              extension.x = 1465545
+              extension.y = 369332
```

Comment:
Something about this makes me uneasy. I know you're supplying default values here that will pass the test, but it seems like the appropriate place for those values is _in_ the spec rather than here in the class itself. I know it would mean passing more information into the textbox element here, but adding an origin (`offset`) and height and width (extent) don't seem _too_ onerous for a consuming client to have to provide to a text box. Maybe I'm jumping the gun, though, and that's part of a future refactor (though that will require changing the test itself to specify values in that case). I guess as default values, these make sense for your specific test, but I'm not sure they make sense for the general consumer of the gem...